Tests/PAML/gen_results.py

```diff
@@ -79,28 +79,28 @@ def baseml(vers=None, verbose=False):
                             test[0] == "nparK" and n in [3, 4]):
                         continue
                     print("\t\tn = {0}".format(n))
-                    ctl_file = os.path.join("Control_files", "baseml",
-                        "{0}{1}.ctl".format(test[0], n))
+                    ctl_file = (os.path.join("Control_files", "baseml",
+                                "{0}{1}.ctl".format(test[0], n)))
```

Comment:
Does fixing the indentation alone (without adding brackets) make the second line too long?